inc/features/blocks/personalization/components/goal-picker.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+const { SelectControl } = wp.components;
+const { __ } = wp.i18n;
+
+const registeredGoals = window.Altis.Analytics.Experiments.Goals || {};
+
+const GoalPicker = ( { goal, onChange } ) => {
+
+    const goals = Object.entries( registeredGoals );
```

Comment:
I can barely read the code for all the linting errors in this block 😬

Reply:
Sorry about that - I need to set up my default formatters etc.. again in vscode after updating to big sur. I didn't notice as I usually have the whitespace indicator turned on.